conf.yaml

```diff
@@ -1345,7 +1369,7 @@ contents:
                   -
                     repo:   ecs-logging
                     path:   docs
-                    map_branches: &mapEcsLoggingJavaToEcsLogging
```

Comment:
Why did this one disappear instead of moving to the top?

Reply:
I assume because it's only used once and therefore the attribute isn't necessary?

Reply:
Ah, that makes sense. Thanks!

Reply:
Yep, there were a couple that were only used once, so I just removed them. I thought about including them anyway, but then realized there was no reason to not make all of the lists into variables.